ami/flowchart/library/Editors.py

```diff
@@ -109,9 +107,6 @@ def state_changed(self, *args, **kwargs):
 
         self.update_plot()
 
-        if self.node:
-            self.node.sigStateChanged.emit(self.node)
-
```

Comment:
We shouldnt emit the signal every time a change is made on the trace editor. This is what is causing the loop when you restore/redisplay. 

There is already an apply button for the trace editor, so just wait until we click apply and emit one signal at the end.